src/app/services/authors-store.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+import { Author } from './author';
+import { AuthorsService } from './authors.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthorsStoreService {
+  private authors$$ = new BehaviorSubject(this.getAllAuthors);
+  private isLoading$$ = new BehaviorSubject(false);
+  public authors$ = this.authors$$.asObservable();
+  public isLoading$ = this.isLoading$$.asObservable();
+
+  constructor(private authorsService: AuthorsService) { }
+
+  getAllAuthors() {
+    return this.authorsService.getAll();
+  }
```

Comment:
```suggestion
  private getAllAuthors() {
    this.isLoading$$.next(true);
    return this.authorsService.getAll().pipe(
        finalize(() => {
            this.isLoading$$.next(false);
        })
    );
  }
```
It is a one of possible ways to make `isLoading$$` work

Reply:
thanks. Done